src/client-scopes/details/ScopeForm.tsx

```diff
@@ -78,7 +78,11 @@ export const ScopeForm = ({ clientScope, save }: ScopeFormProps) => {
         helperTextInvalid={t("common:required")}
       >
         <TextInput
-          ref={register({ required: true })}
+          ref={register({
+            required: true,
+            validate: (value: string) =>
+              !!value.trim() || t("common:required").toString(),
```

Comment:
Why is `.toString()` needed here? Should the return value not always be a string?
```suggestion
              !!value.trim() || t("common:required"),
```